tests/stub/versions/scripts/v5x1_return_1.script

```diff
@@ -0,0 +1,18 @@
+!: BOLT 5.1
```

Comment:
why not `BOLT #VERSION#` here?

Reply:
The siblings scripts doesn't expect the version be send as parameters and neither original code for this test, since this versions script are version specific.